src/files.ts

```diff
@@ -1,10 +1,29 @@
+/* eslint-disable n/no-unsupported-features/node-builtins */
+import assert from "node:assert";
+import { createHash } from "node:crypto";
+import { createReadStream } from "node:fs";
+import { readdir, stat } from "node:fs/promises";
+import { join } from "node:path";
+import { Readable } from "node:stream";
+import { pipeline } from "node:stream/promises";
+
 import { FileLike, filesFromPaths } from "files-from-path";
 
+interface originalInfo {
+  name: string;
+  hash: string;
+  size: number;
+}
+
 export interface SplitFileLike extends FileLike {
-  originalName: string;
-  offset: number;
+  media_type?: string;
+  originalInfo?: originalInfo;
 }
 
+// We split files between CARs if the files are larger than this % of the desired
+// car size.
+const splitFileIfOverPercentage = 5;
```

Comment:
That seems far too low. Is this because splitting into lots of small parts makes it easier to pack very efficiently?

If I've got that right I think it's the wrong trade-off, since efficient downloading of the whole original is important, and splitting large files into lots of parts and spreading it across lots of pieces will become problematic.

Reply:
Please suggest the number you would like here instead.

Reply:
I'd start with 50

Reply:
updated, thanks